src/components/Fence/FenceAuthorizedStudies.js

```diff
@@ -105,64 +108,62 @@ const Gen3ProjectList = compose(withApi, withTheme, withHistory)(
           }
         }
       `}
-      variables={projectIds.reduce(
-        (acc, id) => ({
-          ...acc,
-          [`${toGqlString(id)}_sqon`]: sqonForStudy(id),
-        }),
-        {},
-      )}
-      render={({ loading, data }) => {
-        const aggregations = get(data, 'file');
-        return aggregations ? (
-          projectIds
-            .filter(id =>
-              get(
-                aggregations,
-                `${toGqlString(id)}.participants__study__short_name.buckets.length`,
-              ),
-            )
-            .map(id => {
-              const studyNameBuckets = get(
-                aggregations,
-                `${toGqlString(id)}.participants__study__short_name.buckets`,
-              );
-              const studyName = studyNameBuckets[0];
-              const sqon = sqonForStudy(id);
-              return (
-                <ItemRowContainer>
-                  <Column justifyContent="center" p={15}>
-                    <StackIcon width={20} />
-                  </Column>
-                  <Column flex={1} justifyContent="center" pr={10}>
-                    <Span>
-                      <strong>{studyName ? `${studyName.key} ` : ''}</strong>({id})
+    variables={projectIds.reduce(
+      (acc, id) => ({
+        ...acc,
+        [`${toGqlString(id)}_sqon`]: sqonForStudy(id),
+      }),
+      {},
+    )}
+    render={({ loading, data }) => {
+      const aggregations = get(data, 'file');
+      return aggregations ? (
+        projectIds
+          .filter(id =>
+            get(aggregations, `${toGqlString(id)}.participants__study__short_name.buckets.length`),
+          )
+          .map(id => {
+            const studyNameBuckets = get(
+              aggregations,
+              `${toGqlString(id)}.participants__study__short_name.buckets`,
+            );
```

Comment:
Do we need a default value in this `get` as a third param?

Reply:
We do, get returns 'undefined' if there is no value, and we immediately assume an array on the next line

Nice catch, I just copied this over from an existing element...

Reply:
👍